README.md

```diff
@@ -219,11 +219,12 @@ POST [/\<groepering>]*/\<event> waarbij \<event> eindigt op een voltooid deelwoo
 -   Error model (gebaseerd op RFC7807)
 ```
 {
-    "type": "string" (verplicht, URIformaat),
-    "title": "string" (verplicht),
-    "status": number (verplicht, HTTP response code),
-    "identifier": "string" (verplicht, uniek),
-    "code": "string" (verplicht, foutcode voor ontwikkelaars)
+    "type": "string"                           (verplicht, URIformaat),
+    "title": "string"                          (verplicht),
+    "status": number                           (verplicht, HTTP response code),
+    "identifier": "string"                     (verplicht, uniek),
+    "code": "string"                           (verplicht, foutcode voor ontwikkelaars),
+    "eventuele_extra_parameters": "custom"     (optioneel, bv een array van validatiefouten)
```

Comment:
in het Engels zoals de andere , extra_parameters (eventueel mag weg, wordt aangegeven door de optionaliteit)